acceptance_tests/cypress/integration/lotse_flow.spec.js

```diff
@@ -651,12 +651,6 @@ context('Acceptance tests', () => {
                 expect(loc.pathname.toString()).to.contain('/lotse/step/vorsorge');
             });
 
-            // Redirect gem_haushalt
-            cy.visit('/lotse/step/gem_haushalt?link_overview=True')
-            cy.location().should((loc) => {
-                expect(loc.pathname.toString()).to.contain('/lotse/step/familienstand');
-            });
-
```

Comment:
I am wondering why we were able to keep the assertion in line 671 but not this one here.

I agree with you that the unit tests do test that part quite well. Nevertheless, I would propose that we try to keep this one updated and then change it in one go to make sure that we will still understand everything that happens here? This would mean that we should be still explicit here at what point we expect a redirect and where we do not.